dietpi/dietpi-software

```diff
@@ -6718,6 +6725,111 @@ _EOF_
 
 		fi
 
+		software_id=204 # Navidrome
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 ))
+		then
+
+			Banner_Installing
+
+			# ARMv6
+			local arch='armv6'
+
+			# ARMv7
+			if (( $G_HW_ARCH == 2 )); then
+				arch='armv7'
+
+			# ARMv8
+			elif (( $G_HW_ARCH == 3 )); then
+				arch='arm64'
+
+			# x86_64
+			elif (( $G_HW_ARCH == 10 )); then
+				arch='x86_64'
+
+			fi
```

Comment:
Let's reduce the line count a little:
```suggestion
			case $G_HW_ARCH in
				1) local arch='armv6'
				2) local arch='armv7'
				3) local arch='arm64'
				*) local arch='x86_64'
			esac
```

Reply:
looks like `Shellcheck` don't like our optimization 🤣